help/blocks/search/search.css

```diff
@@ -34,22 +132,61 @@
     font-size: 1.125rem;
     line-height: 1.5rem;
   }
+
+  .search .pagination {
+    margin: 0 1.5rem;
+  }
+
+  .search .results-list {
+    margin: 0 1.5rem 1rem;
+  }
+
+  .search .error-messages {
+    margin: 0.25rem 1rem;
+  }
+
 }
 
 @media (min-width: 900px) {
   .search-input {
-    padding: 24px 56px 24px 24px;
+    padding: 1.5rem 3.5rem 1.5rem 1.5rem;
```

Comment:
I see extra white spaces in bottom and also greyish background colour in the top(which should not be present). Maybe we should fix these.